packages/aura-language-server/CONTRIBUTING.md

```diff
@@ -16,21 +16,7 @@ npm test -- --watch         # Run the test in watch mode
 
 # Running
 
-To run/debug this language server in VSCode you can launch it from the [salesforcedx-vscode](https://github.com/midzelis/salesforcedx-vscode) client extension (Note: this is currently points to Min's fork until we merge our stuff into the DX extension)
```

Comment:
This is old out of date documentation. When performing QAs for the lightning language server, the steps listed from the README worked as expected.